src/main/java/easy/mind/com/api/сontroller/TestsQuestionsForAdminController.java

```diff
@@ -0,0 +1,29 @@
+package easy.mind.com.api.сontroller;
+
+import easy.mind.com.api.DTO.TestsQuestionsDTO;
+import easy.mind.com.api.facade.TestsQuestionsFacade;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/questions")
+@RequiredArgsConstructor
+public class TestsQuestionsForAdminController {
+    private final TestsQuestionsFacade testsQuestionsFacade;
+
+    @PostMapping
+    public ResponseEntity<TestsQuestionsDTO> createQuestions(@Valid @RequestBody TestsQuestionsDTO testsQuestionsDTO) {
+        return ResponseEntity.status(HttpStatus.CREATED)
+                .body(testsQuestionsFacade.create(testsQuestionsDTO));
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<TestsQuestionsDTO> deleteQuestion(@PathVariable int id, @PathVariable int userId) {
```

Comment:
UserId is not used there. Moreover, we don't have it in path.

Reply:
Did you even tested your changes? ;)

Reply:
please don't be toxic